datalad_next/patches/push_to_export_remote.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from typing import (
+    Dict,
+    Generator,
+    Iterable,
+    Optional,
+    Union,
+)
+from unittest.mock import patch
+
+import datalad.core.distributed.push as push
+from datalad.distribution.dataset import Dataset
+from datalad.runner.exception import CommandError
+from datalad.support.annexrepo import AnnexRepo
+from datalad.support.exceptions import CapturedException
+from datalad_next.credman import CredentialManager
+from datalad_next.utils import (
+    get_specialremote_credential_envpatch,
+    get_specialremote_credential_properties,
+    needs_specialremote_credential_envpatch,
+)
+
+
+__docformat__ = "restructuredtext"
+
+
+lgr = logging.getLogger('datalad_next.push_to_export_remote')
+
+
+def _is_export_remote(remote_info: Optional[Dict]) -> bool:
+    """Check if remote_info is valid and has exporttree set to "yes"
+
+    :param remote_info: None or a dictionary the contains git annex special
+    remote info
+    :return: True if exporttree is yes, else False
```

Comment:
Here and elsewhere, we should stick to the numpydoc conventions

Reply:
ok, will change it